### ISSUE TITLE
Restrict species list available to each organization

### ISSUE THREAD

**Issue by nmcharlton** [Greenstand/treetracker-admin-client#232]
Currently, all users see the same list of species when tagging or filtering captures.
Several of these species names are specific to organizations or in some other way not relevant to all users.

It should be possible to restrict the organizations to which each species is available.
The Greenstand Species Manager should have the ability to select which species are available to each organization (requires some UI/UX design).
Users in that organization would then only see the species assigned to them in the tag and filter lists.

This requires a way of linking a species to one or more organizations in the database and potentially a new microservice to manage species.
This should be implemented within the proposed domain model below and be linked to the new Stakeholder microservice/schema:

![image](https://user-images.githubusercontent.com/5558838/144754417-86170471-e0dc-4dbc-ac75-0a54daa29977.png)

**Comment by sebastiangaertner** [comment]
This is becoming more and more pressing since there are a lot of Species in the drop down already and new organisations using the admin panel are totally confused.

**Comment by camwebb** [comment]
The only way to solve this easily is via country. First find the country of the organization and then use the `<planted_in/>` attribute in the [XML file](https://github.com/Greenstand/Tree_Species/blob/master/tree_species.xml). It would be too much work to keep track at the level of organization.

**Comment by sebastiangaertner** [comment]
Well as i mentioned the organizations should know what they plant so they will have to send in their species list as part of the onboarding process or?

**Comment by camwebb** [comment]
Currently there's no required submission of species lists for an organization. Wish there was - it would make the taxonomy work easier. Species lists requests are being documented in the `#species_requests` slack channel, but it is hit-or-miss if we get any info back.

**Comment by sebastiangaertner** [comment]
That is the plan. An organization should be able to choose from the total pool of species and also be able to request species to be added. When requesting a species it needs to be displayed in a list and reason for rejection stated or the link to the new species next to the newly added species.

**Comment by nmcharlton** [comment]
See Greenstand/treetracker-decisions#21 for the updated schema proposition.

**Comment by sebastiangaertner** [comment]
Yes, thank you Nick. @camwebb can we dupdate the schema diagrams to reflect the next level herbarium API? That was the UX design can get started.

**Comment by camwebb** [comment]
@sebastiangaertner Not sure what your question is. My suggested schema is [here](https://github.com/Greenstand/treetracker-decisions/blob/main/0012-species-data.md), and [here](https://github.com/Greenstand/Tree_Species/blob/master/sql/new_species_info.sql) (in SQL).
